packages/@n8n/decorators/src/__tests__/redactable.test.ts

```diff
@@ -0,0 +1,195 @@
+import { UnexpectedError } from 'n8n-workflow';
+
+import { Redactable, RedactableError } from '../redactable';
+
+describe('Redactable Decorator', () => {
+	class TestClass {
+		@Redactable()
+		methodWithUser(arg: {
+			user: { id: string; email?: string; firstName?: string; lastName?: string; role: string };
+		}) {
+			return arg;
+		}
+
+		@Redactable('inviter')
+		methodWithInviter(arg: {
+			inviter: { id: string; email?: string; firstName?: string; lastName?: string; role: string };
+		}) {
+			return arg;
+		}
+
+		@Redactable('invitee')
```

Comment:
Would be nice to test having 2 decorators, like it's used here: https://github.com/n8n-io/n8n/blob/master/packages/cli/src/events/relays/log-streaming.event-relay.ts#L277
(By the way, out of scope, but maybe accepting multiple fieldNames might be more redable and less prone to interpretation on how the decorators will affect being called 2 times on the same args

Reply:
Let's do that in this same PR. I'll take care of it.